05/app.js

```diff
@@ -0,0 +1,61 @@
+const form = document.querySelector("form");
+const msgList = document.querySelector("ul");
+let msgArray = [];
+form.addEventListener("submit", handleSubmit);
+
+function handleSubmit(e) {
+  e.preventDefault();
+  const firstName = form.elements[0].value;
+  const lastName = form.elements[1].value;
+  const street = form.elements[2].value;
+  const houseNumber = form.elements[3].value;
+  const zipCode = form.elements[5].value;
+  const city = form.elements[6].value;
+  const voivodeship = form.elements[7].value;
+
+  const error = [];
+  const regex = new RegExp("^[0-9]{2}-[0-9]{3}$");
+
+  if (firstName.trim() === "") {
```

Comment:
Poniżej robi się rozbudowana lista if-ów.
Warto zrobić z tego bardziej uniwersalne rozwiązanie.
Postaraj się to poprawić wzorując się na tym artykule:  https://devmentor.pl/b/walidacja-formularza-w-javascript